README.md

```diff
@@ -4,7 +4,7 @@ Pulse.js is a standalone JavaScript function queue object that is mainly used to
 Pulse.js is mainly used for animation queuing, but can easily be adapted to any other synchronous function firing systems.
 
 ## Installation
-Simply include the jaz production file on your webpage (preferably all webpages).
+Simply include the pulse production file on your webpage (preferably all webpages).
```

Comment:
Avoid trivializing-language like "simply"

```suggestion
Include the pulse production file on your webpage (preferably all webpages).
```

Reply:
I think now if I reply, the bug occurs?

Reply:
What if I hop in

Reply:
no i gotta jump in

Reply:
should b 2 now i think

Reply:
should not see this

Reply:
Another comment for you 🙂